### ISSUE TITLE
Desktop notification

### ISSUE THREAD

**Issue by grisu48** [os-autoinst/openqa-mon#5]
Would be nice to have desktop notifications, when in continuous mode.

**Comment by pdostal** [comment]
Here is the [terminal bell](https://www.rosettacode.org/wiki/Terminal_control/Ringing_the_terminal_bell) character idea:
```go
package main
 
import "fmt"
 
func main() {
  fmt.Print("\a")
}
```

**Comment by grisu48** [comment]
Thanks. The terminal bell might be the easiest to implement as a starter.

**Comment by grisu48** [comment]
The terminal beep seem to not work anymore on modern systems. Stackoverflow and Launchpad are full of related discussions.

**Comment by grisu48** [comment]
Implemented bell and desktop notifications in #6
